### ISSUE TITLE
Scenario Editor Hitting rule endpoint repeatedly

### ISSUE THREAD

**Issue by Gauntlet173** [Lexpedite/blawx#464]
Scenario editor is hitting the rule endpoint repeatedly, even after answers for those questions should be cached. It's possible that there is a sync/async thing going on, but it feels like the ultimate solution to this problem is to pre-generate the needed text and supply it as part of the reasoner's response, or as part of the ontology response, so as to ensure it happens only once, and quickly.

**Comment by Gauntlet173** [comment]
I discovered that the problem here was actually how slow the rule endpoint was, for no reason. So this is being kept (because I think it still may be hitting the rule endpoint more than is necessary), but it's a low priority because it's not causing any noticeable delay anymore.
